### DISCUSSION TITLE
Edges over nodes and not through

### DISCUSSION THREAD

**Discussion by gitHoussem** [bcakmakoglu/vue-flow#1094] [Q&A] [answered]
Hi, 

Is there a way to configure the smooth step edge to (try) and go over the nodes and not through the nodes ? 

Here's an example : 

![image](https://github.com/bcakmakoglu/vue-flow/assets/43743206/b10a0ee6-2131-4e76-b10f-4853b40d9319)

Here is the desired result: 

![image](https://github.com/bcakmakoglu/vue-flow/assets/43743206/e89f040c-51b2-4350-b478-a02c6b81246e)

I tried changing the destination position to be top, but sometimes it does the following : 

![image](https://github.com/bcakmakoglu/vue-flow/assets/43743206/e367fa74-eb24-4399-b20b-0b6ccb36257b)

Thank you!

**Comment by bcakmakoglu** [comment]
Late response but:
You can use `edge.zIndex` to elevate a specific edge *or* just use css to elevate the whole edge containers

```css
.vue-flow__edges {
	z-index: 9999;
}
```

Make sure to use an index that's something over 1000 as nodes get elevated to *at least* 1000 when selected.

**Reply by gitHoussem** [reply]
Hey, thanks for the answer. 
However this would bring the edge **on top of the node** whereas I wish for it to **go around the node**
I saw a [pending PR for this](https://github.com/bcakmakoglu/vue-flow/pull/998) but I don't believe it supports the default Bezier edge.

**Reply by bcakmakoglu** [reply, marked as answer]
That's a functionality that's not part of the core package, sorry - there is a sort of deprecated package for a smart edge but imo it doesn't work well enough to be considered production ready by all means.

I'll have to disappoint you on this one.

**Reply by gitHoussem** [reply]
I see! I appreciate the answer
